<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>911</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QLabel" name="camera_set">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>591</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="x1_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="current_value">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X1 Y1 -&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="y1_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="current_value2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>110</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X2 Y2 -&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="x2_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="y2_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>160</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="current_value3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>200</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X3 Y3 -&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="x3_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>220</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="y3_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>250</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="current_value4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>290</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X4 Y4 -&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="x4_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>310</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="y4_slider">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>340</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>700</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CardTab">
    <attribute name="title">
     <string>Card</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>401</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSlider_3">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Low S</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Low V</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Low H</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>179</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_3">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>401</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSlider" name="horizontalSlider_4">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSlider_5">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider_6">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_4">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_5">
        <property name="maximum">
         <number>179</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_6">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Up H</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Up V</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Up S</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="frame_card2">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>12</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="frame_card1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Red</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Green</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Blue</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Yellow</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Black</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLabel" name="card_output">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="card_output_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>189</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>ON Cam</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stop_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>OFF Cam</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>561</y>
     <width>751</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>570</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect Serial</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect Serial</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>863</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>853</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
